### DISCUSSION TITLE
Wix 5 Toolset Samples

### DISCUSSION THREAD

**Discussion by guenhter** [wixtoolset/issues#8568] [Questions] [answered]
Hi,

I'm completely new to Wix and I find it really hard to get any resources how to create the Xml file and use the wix toolset. 
I don't use VisualStudio or msbuild but instead VSCode and some fancy langauge where a .exe pops out. Creating an installer for this shouldn't be the hardest thing in the world, but without proper instructions it becomes a challenge. 

I'd be very thankful for any links or thelinke to use the latest Wix 5 toolset.

**Comment by robmen** [comment]
I'm curious what links you've already read.

**Comment by guenhter** [comment]
I basically Read

* https://wixtoolset.org/docs/intro/

and everything under

* https://wixtoolset.org/docs/tools/

**Reply by robmen** [reply, marked as answer]
Cool. Useful information.

You might check out the [FireGiant Tutorial](https://www.firegiant.com/docs/wix/tutorial/). It starts with Visual Studio (since that's the easiest way to get started), but you get to the XML language information pretty quickly after the project creation. I also started my [Deployment Dojo series](https://robmensching.com/deployment-dojo/episodes/s1/) using the command-line which may be informative.

**Reply by guenhter** [reply]
The video was super helpful. After watching the it, also the linked Tutorial becomes cristal clear. I'm sure for someone who is already into the whole Wix Toolset, the current documentation is perfectly understandable and very helpful. Here some Feedback from my end about how new-starters could be onboarded easier:

* In the Getting Started Page (https://wixtoolset.org/docs/intro/), for other projects it turned out very helpful to not only have the installation there but also a minimal working example. In your case, I would just use the example with the "a.txt" file of the video with the 16 lines of XML. No explaination about packages, features, ... but just the XML and the command to build it. This is extremely satisfying if one copies a few XML lines, runs a `wix build`, executes  the produced MSI and this thing shows up under the "installed programs" in Windows. And I guess it is not much work to be added to the documentation :)
* The step-by-step instructions of the linked tutorial (https://www.firegiant.com/docs/wix/tutorial/) is great to have. It explains the details of every line. What would be very helpful here as well is a section at the bottom "putting it all together" where the whole XML shows up. Here is a great example of such a tutorial: https://dev.to/theembeddedrustacean/stm32f4-embedded-rust-at-the-pac-gpio-control-20h4  I used that tuturial and the "Full Application Code" section is extremely helpufl.


Thx for your fast reaction on this matter. Really appreciate that.

**Comment by robmen** [comment]
We generally keep Discussions open so that they are more easily found by search.
